packages/client/src/pages/CarbonHackCards/ForecastCard/ForecastCard.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { render, within } from '@testing-library/react'
+import ForecastCard from './ForecastCard'
+import { mockData } from '../../../utils/data/mockData'
+import useRemoteForecastService from '../../../utils/hooks/ForecastServiceHook'
+import { forecastServiceMockReturnValue } from './ForecastCardMockData'
+import { ServiceResult } from '../../../Types'
+import { ForecastResult } from '../../../utils/hooks/ForecastServiceHook'
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
Let's add a new line to separate the imports from the begging in the variable declarations